spec/lib/vcr/errors_spec.rb

```diff
@@ -102,7 +110,15 @@ def request_with(values)
           end
         end
 
-        it 'mentions :allow_playback_repeats if the current cassette has a used matching interaction' do
+        it 'does not mention the :once or :none record modes if using the :new_episodes record mode at least in one cassette' do
```

Comment:
Line is too long. [128/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.